101-azurefirewall-create-with-zones/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetworkName": {
```

Comment:
remove this when you add the defaultValue

Reply:
Done

Reply:
Done